examples/netint/netint_example.c

```diff
@@ -54,20 +54,20 @@ void create_format_strings(const EtcPalNetintInfo* netint_arr, size_t num_interf
     etcpal_inet_ntop(&netint->addr, addr_str, ETCPAL_INET6_ADDRSTRLEN);
     etcpal_inet_ntop(&netint->mask, netmask_str, ETCPAL_INET6_ADDRSTRLEN);
 
-    size_t name_len = strlen(netint->name);
+    size_t id_len = strlen(netint->id);
     size_t addr_len = strlen(addr_str);
     size_t netmask_len = strlen(netmask_str);
 
-    if (name_len > longest_name)
-      longest_name = name_len;
+    if (id_len > longest_name)
+      longest_name = id_len;
```

Comment:
i think it would make sense to change longest_name to longest_id to keep it consistent with the change from name to id

Reply:
Fixed! Thanks!